LogMonitor/src/LogMonitor/EventMonitor.cpp

```diff
@@ -710,3 +711,43 @@ EventMonitor::EnableEventLogChannel(
         EvtClose(channelConfig);
     }
 }
+
+/// <summary>
+/// XML Formatted Event
+/// </summary>
+std::wstring EventMonitor::XMLFormattedEvent()
+{
+    auto logFmt = L"<Source>%s</Source><LogEntry><Time>%s</Time><Channel>%s</Channel><Level>%s</Level><EventId>%u</EventId><Message>%s</Message></LogEntry>";
+    std::wstring formattedEvent = Utility::FormatString(
+        logFmt,
+        source.c_str(),
+        eventTime.c_str(),
+        eventChannel.c_str(),
+        eventLevel.c_str(),
+        eventId,
+        eventMessage.c_str()
+    );
+
+    return formattedEvent;
+}
+
+/// <summary>
+/// JSON Formatted Event
+/// </summary>
+std::wstring EventMonitor::JSONFormattedEvent()

```

Comment:
I don't know what happened to my previous comment on this. I'd suggested that since these two methods are simple, might be succinct to just do away with them and have modify ~L564 to:

```cpp
auto logFmt = L"<Source>%s</Source><LogEntry><Time>%s</Time><Channel>%s</Channel><Level>%s</Level><EventId>%u</EventId><Message>%s</Message></LogEntry>";
if (Utility::CompareWStrings(m_logFormat, L"JSON")) {
   logFmt = L"{\"Source\": \"EventLog\",\"LogEntry\": {\"Time\": \"%s\",\"Channel\": \"%s\",\"Level\": \"%s\",\"EventId\": %u,\"Message\": \"%s\"}}";
}
std::wstring formattedEvent = Utility::FormatString(
    logFmt,
    source.c_str(),
    eventTime.c_str(),
    eventChannel.c_str(),
    eventLevel.c_str(),
    eventId,
    eventMessage.c_str()
);

logWriter.WriteConsoleLog(formattedEvent);
```

Reply:
This has been addressed.